website/components/info.tsx

```diff
@@ -106,99 +105,109 @@ export default function Info() {
+| | | |  __/\\__ \\ |_ 
+|_| |_|\\___||___/\\__|
+`}
+                  </pre>
+                  <div className="grid grid-cols-1 sm:grid-cols-2 gap-6 w-full">
+                    <div className="flex items-center">
+                      <GoCpu className="text-3xl sm:text-4xl text-HCPurpleText mr-3" />
+                      <div>
+                        <div className="text-sm font-medium">Processor</div>
+                        <div className="text-xs text-gray-300">i5-13500 (14 cores, 20 threads)</div>
+                      </div>
+                    </div>
+                    <div className="flex items-center">
+                      <BsMemory className="text-3xl sm:text-4xl text-HCPurpleText mr-3" />
+                      <div>
+                        <div className="text-sm font-medium">Memory</div>
+                        <div className="text-xs text-gray-300">64 GB DDR4 RAM</div>
+                      </div>
+                    </div>
+                    <div className="flex items-center">
+                      <BsNvme className="text-3xl sm:text-4xl text-HCPurpleText mr-3" />
+                      <div>
+                        <div className="text-sm font-medium">Storage</div>
+                        <div className="text-xs text-gray-300">2x 512GB NVMe SSD & 1x 1TB NVMe SSD</div>
+                      </div>
+                    </div>
+                    <div className="flex items-center">
+                      <BsHddNetwork className="text-3xl sm:text-4xl text-HCPurpleText mr-3" />
+                      <div>
+                        <div className="text-sm font-medium">Network</div>
+                        <div className="text-xs text-gray-300">1 Gbps, IPv4 & IPv6</div>
+                      </div>
+                    </div>
+                  </div>
+                  <div className="mt-4 text-sm">
+                    <p>Nest runs on Proxmox VE 8.1.3 with two main VMs:</p>
+                    <ul className="list-disc list-inside mt-2">
+                      <li>Secure VM (NixOS): Hosts critical services</li>
+                      <li>Nest VM (Debian 12): User-accessible environment</li>
+                    </ul>
+                    <p className="mt-2">
+                      <a href="https://github.com/hackclub/nest/blob/main/SETUP.md" className="text-HCPurpleText hover:underline">
+                        Read more about Nest&apos;s technical setup on GitHub
+                      </a>
+                    </p>
+                  </div>
+                </div>
+              ) : (
+                <motion.div
+                  initial={{ height: 0 }}
+                  animate={{ height: "auto" }}
+                  exit={{ height: 0 }}
+                  transition={{ duration: 0.5 }}
+                >
+                  <div className="font-mono text-green-400 mb-4">
+                    <span className="text-blue-400">nest@hackclub:~$</span> cat {component?.toLowerCase()}.txt
```

Comment:
Have a default filename on page load so that it doesn't just say `cat .txt`, maybe `welcome.txt`?

Reply:
Also done, by default it just says "Click on a tab to learn more about Nest!"